data/data/aws/bootstrap/variables.tf

```diff
@@ -18,14 +18,14 @@ variable "cluster_name" {
   description = "The name of the cluster."
 }
 
-variable "elbs" {
+variable "target_group_arns" {
   type        = "list"
   default     = []
   description = "Elastic load balancer IDs to attach to the bootstrap node."
```

Comment:
Should this description be updated to talk about target groups?

Reply:
:+1: